### PR TITLE
feat: Add kubtest-dashboard.

### PR DESCRIPTION
**Pull request by nicufk**
This PR adds kubtest-dashboard to be able to add it on install.
Task: https://github.com/kubeshop/kubtest/issues/341 

## Changes

-

## Fixes

-

## Checklist

- [ ] tested locally
- [ ] added new dependencies
- [ ] updated the docs
- [ ] added a test

### CODE REVIEW

**Review by Tarick — COMMENTED** [review]
Looks OK but a bit unrelated to Helm, how does dashboard JS code knows about API endpoints? This needs to be configurable per user deployment and AFAIK dashboard is built once into static files, there is no server that can get HOST header and rewrite JS to use that Host header.
I see this situation a lot, so I'd suggest adding to Dashboard Dockerfile the Entrypoint script that looks up environment variables for e.g. KUBETEST_API_ENDPOINT and rewrites endpoint in JS file (e.g. index.js.tmpl into index.js), that will be imported by other JS files.

File: kubtest-dashboard/charts/kubtest-dashboard/Chart.yaml [hunk 1]
```diff
@@ -0,0 +1,24 @@
...
+#
+# Library charts provide useful utilities or functions for the chart developer. They're included as
+# a dependency of application charts to inject those utilities and functions into the rendering
+# pipeline. Library charts do not define any templates and therefore cannot be deployed.
+type: application
+
+# This is the chart version. This version number should be incremented each time you make changes
+# to the chart and its templates, including the app version.
+# Versions are expected to follow Semantic Versioning (https://semver.org/)
+version: 0.5.0
```
Comment (jasmingacic) [inline kubtest-dashboard/charts/kubtest-dashboard/Chart.yaml:18]: aren't we going with 0.6.0

Comment (nicufk) [reply kubtest-dashboard/charts/kubtest-dashboard/Chart.yaml:18]: yes, but until we get there I expect to be some fixes.

**Comment by nicufk** [comment]
> Looks OK but a bit unrelated to Helm, how does dashboard JS code knows about API endpoints? This needs to be configurable per user deployment and AFAIK dashboard is built once into static files, there is no server that can get HOST header and rewrite JS to use that Host header. I see this situation a lot, so I'd suggest adding to Dashboard Dockerfile the Entrypoint script that looks up environment variables for e.g. KUBETEST_API_ENDPOINT and rewrites endpoint in JS file (e.g. index.js.tmpl into index.js), that will be imported by other JS files. And then in Helm code add environment variable with endpoint to dashboard pod.

Good point, I was going to do that, for now we can specify the endpoint in the query params or there is a field in the JS app where we can specify the endpoint. Anyway, I added the env var as well

**Review by jasmingacic — APPROVED** [review]

**Review by Tarick — APPROVED** [review]

File: kubtest-dashboard/charts/kubtest-dashboard/templates/tests/test-connection.yaml [hunk 2]
```diff
@@ -0,0 +1,15 @@
+apiVersion: v1
+kind: Pod
```
Comment (fog1985) [inline kubtest-dashboard/charts/kubtest-dashboard/templates/tests/test-connection.yaml:2]: Hey @nicufk ,
What is the purpose of this POD?
What will happen if it fails for example?

**Review by fog1985 — APPROVED** [review]
LGTM

File: kubtest-dashboard/charts/kubtest-dashboard/templates/tests/test-connection.yaml (hunk 2 above)
Comment (nicufk) [reply kubtest-dashboard/charts/kubtest-dashboard/templates/tests/test-connection.yaml:2]: It was generated by helm create.

**Pull request merged**